pkg/cmd/server/origin/admission/plugin_initializer.go

```diff
@@ -99,12 +101,15 @@ func NewPluginInitializer(
 		return nil, nil, err
 	}
 
-	quotaRegistry := oquota.NewAllResourceQuotaRegistryForAdmission(
-		informers.GetExternalKubeInformers(),
+	// TODO make a union registry
```

Comment:
@derekwaynecarr can you review pls?

Reply:
this looks fine, is it accurate to say that this registry here just knows how to evaluate all kube types + the image types and in practice the core kube types will be ignored?